guides/v2.0/config-guide/bootstrap/mage-profiler.md

```diff
@@ -31,13 +31,13 @@ Magento provides the base functionality in <a href="{{ site.mage2000url }}lib/in
 
 	You can also use one of the following values to enable a specific profiler:
 
-    - `csvfile` which uses <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Csvfile.php" target="_blank">Magento\Framework\Profiler\Driver\Standard\Output\Csvfile</a>
-    - Any other value (except `2`), including an empty value, which uses <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Html.php" target="_blank">Magento\Framework\Profiler\Driver\Standard\Output\Html</a>
+    - `csvfile` which uses [Magento\\Framework\\Profiler\\Driver\\Standard\\Output\\Csvfile]({{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Csvfile.php){: target="_blank"}
```

Comment:
Not supposed to be two `\` and I don't think there is supposed to be a space between `:` and `target` here.

Reply:
Same for below